src/m1_facilitator/SeedDataInitializer.ts

```diff
@@ -70,23 +74,39 @@ export default class SeedDataInitializer {
     );
 
     const originAnchorAddress = await erc20Gateway.methods.stateRootProvider().call();
-    Logger.debug(`originAnchorAddress: ${originAnchorAddress}`);
-    const originAnchorInstance = Mosaic.interacts.getAnchor(
-      originWeb3,
-      originAnchorAddress,
-    );
-
     const auxiliaryAnchorAddress = await erc20Cogateway.methods.stateRootProvider().call();
-    Logger.debug(`auxiliaryAnchorAddress: ${auxiliaryAnchorAddress}`);
-    const auxiliaryAnchorInstance = Mosaic.interacts.getAnchor(
-      auxiliaryWeb3,
-      auxiliaryAnchorAddress,
-    );
+    let auxiliaryLatestAnchoredStateRootBlockHeight: string;
+    let originLatestAnchoredStateRootBlockHeight: string;
+    if (ArchitectureLayout.MOSAIC_0_14_GEN_1 === architectureLayout) {
```

Comment:
Can you add logs that show which part of if/else is triggered?

Reply:
Done